src/invoices/InvoiceLineDetails/OtherRelatedInvoiceLines/OtherRelatedInvoiceLines.js

```diff
@@ -38,11 +45,15 @@ const getResultFormatter = ({ search }) => ({
         search,
       }}
     >
-      {`${invoiceLine.invoiceLineNumber}`}
+      {`${invoiceLine?.invoiceLineNumber}`}
     </Link>
   ),
+  fiscalYear: invoiceLine => invoiceLine.fiscalYear?.code || <NoValue />,
   [COLUMN_INVOICE_DATE]: invoiceLine => <FormattedDate value={invoiceLine.invoice?.invoiceDate} />,
-  vendorName: invoiceLine => invoiceLine.vendor?.name || <NoValue />,
+  vendorCode: invoiceLine => invoiceLine.vendor?.code || <NoValue />,
+  subscriptionStart: invoiceLine => getFormattedDate(invoiceLine.fiscalYear?.periodStart),
```

Comment:
Why don't to use just `<FormattedDate />` as it's already used?

Reply:
Haven't noticed :)